src/main/java/org/wso2/siddhi/plugins/idea/SiddhiTypes.java

```diff
@@ -0,0 +1,161 @@
+/*
+ *  Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.wso2.siddhi.plugins.idea;
+
+import com.intellij.psi.tree.TokenSet;
+import org.antlr.jetbrains.adaptor.lexer.PSIElementTypeFactory;
+import org.antlr.jetbrains.adaptor.lexer.RuleIElementType;
+import org.antlr.jetbrains.adaptor.lexer.TokenIElementType;
+import org.wso2.siddhi.plugins.idea.grammar.SiddhiQLLexer;
+
+import java.util.List;
+
+/**
+ * Represents a token in the language of the plug-in. The "token type" of
+ * leaf nodes in jetbrains PSI tree. Corresponds to ANTLR's int token type.
+ * Intellij lexer token types are instances of IElementType:
+ * <p>
+ * "Interface for token types returned from lexical analysis and for types
+ * of nodes in the AST tree."
+ * <p>
+ * We differentiate between parse tree subtree roots and tokens with
+ * {@link RuleIElementType} and {@link TokenIElementType}, respectively.
+ */
+public class SiddhiTypes {
+
+    private SiddhiTypes() {
+
+    }
+
+    private static final List<TokenIElementType> tokenIElementTypes =
+            PSIElementTypeFactory.getTokenIElementTypes(SiddhiLanguage.INSTANCE);
+
+    public static final TokenIElementType IDENTIFIER = tokenIElementTypes.get(SiddhiQLLexer.ID);
+    public static final TokenIElementType INT_LITERAL = tokenIElementTypes.get(SiddhiQLLexer.INT_LITERAL);
+    public static final TokenIElementType LONG_LITERAL = tokenIElementTypes.get(SiddhiQLLexer.LONG_LITERAL);
+    public static final TokenIElementType FLOAT_LITERAL = tokenIElementTypes.get(SiddhiQLLexer.FLOAT_LITERAL);
+    public static final TokenIElementType DOUBLE_LITERAL = tokenIElementTypes.get(SiddhiQLLexer.DOUBLE_LITERAL);
+    public static final TokenIElementType COL = tokenIElementTypes.get(SiddhiQLLexer.COL);
+    public static final TokenIElementType SCOL = tokenIElementTypes.get(SiddhiQLLexer.SCOL);
+    public static final TokenIElementType DOT = tokenIElementTypes.get(SiddhiQLLexer.DOT);
+    public static final TokenIElementType TRIPLE_DOT = tokenIElementTypes.get(SiddhiQLLexer.TRIPLE_DOT);
+    public static final TokenIElementType OPEN_PAR = tokenIElementTypes.get(SiddhiQLLexer.OPEN_PAR);
+    public static final TokenIElementType CLOSE_PAR = tokenIElementTypes.get(SiddhiQLLexer.CLOSE_PAR);
+    public static final TokenIElementType OPEN_SQUARE_BRACKETS = tokenIElementTypes.get(SiddhiQLLexer
```

Comment:
This would look much better if we change this like below. Fix in all other places as well.

```
public static final TokenIElementType OPEN_SQUARE_BRACKETS = 
        tokenIElementTypes.get(SiddhiQLLexer.OPEN_SQUARE_BRACKETS);
```

Reply:
Fixed in commit 3aa87a6